packages/libs/eda/src/lib/core/components/visualizations/implementations/HistogramVisualization.tsx

```diff
@@ -1540,12 +1540,7 @@ function getRequestParams(
   ) => HistogramRequestParams
 ): HistogramRequestParams {
   const {
-    binWidth = NumberVariable.is(variable) || DateVariable.is(variable)
-      ? config.independentAxisValueSpec === 'Full'
-        ? variable.distributionDefaults.binWidthOverride ??
-          variable.distributionDefaults.binWidth
-        : undefined
-      : undefined,
+    binWidth = undefined,
     binWidthTimeUnit = variable?.type === 'date'
       ? variable.distributionDefaults.binUnits
       : undefined,
```

Comment:
The `= undefined` is great for readability in the diff, but not necessary for functionality.

Furthermore, we don't need to set `binWidthTimeUnit` from the `variable.distributionDefaults` because we will never pass it to the back end in `binSpec` if `binWidth` is falsy.

```suggestion
    binWidth,
    binWidthTimeUnit,
```

The same goes for Lineplot

Reply:
@bobular Yes you are correct. I was thinking of defining it explicitly, rather than implicitly to make it clearer, but we can simply not define it as you suggested. I was also gauging whether binWidthTimeUnit was set to be undefined as well, but I just left it as was 😅 Anyway I will update them.